cryptoki/src/mechanism/mod.rs

```diff
@@ -108,6 +108,20 @@ impl MechanismType {
         val: CKM_SHA512_RSA_PKCS,
     };
 
+    // SHAn-RSA-PKCS-PSS
+    /// SHA256-RSA-PKCS-PSS mechanism
+    pub const SHA256_RSA_PKCS_PSS: MechanismType = MechanismType {
+        val: CKM_SHA256_RSA_PKCS_PSS,
+    };
+    /// SHA384-RSA-PKCS-PSS mechanism
+    pub const SHA384_RSA_PKCS_PSS: MechanismType = MechanismType {
+        val: CKM_SHA384_RSA_PKCS_PSS,
+    };
+    /// SHA512-RSA-PKCS-PSS mechanism
+    pub const SHA512_RSA_PKCS_PSS: MechanismType = MechanismType {
+        val: CKM_SHA512_RSA_PKCS_PSS,
+    };
```

Comment:
Could you also add `SHA1_RSA_PKCS_PSS` for completeness?

Reply:
Done!